index.js

```diff
@@ -61,9 +67,24 @@ module.exports = {
             testCredentials = {};
         }
 
+        if (typeof testFlow === "string") {
+                testFlow = JSON.parse(testFlow);
```

Comment:
this will throw if `testFlow` is a string but not valid JSON.  you will probably want to try / catch and pass the exception to `cb()`.